.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,20 @@
+{
+  "dockerFile": "Dockerfile",
+  "runArgs": [
+    "--privileged"
+  ],
+  "forwardPorts": [
+    2375,
+    2376
+  ],
+  "extensions": [
+    "formulahendry.dotnet-test-explorer",
+    "ms-azuretools.vscode-docker",
+    "ms-dotnettools.csharp"
+  ],
+  "settings": {
+    "omnisharp.path": "latest" // https://github.com/OmniSharp/omnisharp-vscode/issues/5410#issuecomment-1284531542.
+  },
+  "postCreateCommand": "git config --system --add safe.directory ${containerWorkspaceFolder} && nohup /bin/sh -c 'dockerd &'",
```

Comment:
This is a very unusual approach to give Docker support, you normally enable it through devcontainers feature definition:
https://github.com/testcontainers/workshop/blob/main/.devcontainer/devcontainer.json#L12